.github/workflows/mdspan-version-check.yml

```
@@ -1,7 +1,14 @@
 name: mdspan version check
 
 on:
-  workflow_call:
+  push:
+    paths:
+      - 'tpls/mdspan/**'
+    branches:
+      - develop
+  pull_request:
+    paths:
+      - 'tpls/mdspan/**'
```

```suggestion
    paths:
      - '.github/workflows/mdspan-version-check.yml'
      - 'tpls/mdspan/**'
    branches:
      - develop
  pull_request:
    paths:
      - '.github/workflows/mdspan-version-check.yml'
      - 'tpls/mdspan/**'
```

If one modifies the reference commit number, we should also trigger the workflow right ?